server_action_domain/tests/test_server_action_domain.py

```diff
@@ -0,0 +1,72 @@
+# Copyrithg 2020 Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestServerActionDomain(common.TransactionCase):
```

Comment:
pls use SavepointCase and do this in `setUpClass`:

```py
cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
otherwise this code runs for every tests and creates useless records.

Reply:
kind of mandatory :innocent: